doc/prediction.rst

```diff
@@ -146,3 +146,48 @@ instance we might accidentally call ``clf.set_params()`` inside a predict functi
 
     with ThreadPoolExecutor(max_workers=10) as e:
         e.submit(predict_fn, ...)
+
+*****************************
+Privacy-Preserving Prediction
+*****************************
+
+Concrete ML is a third-party open-source library developed by `Zama`_ that proposes gradient
+boosting classes similar to ours, but predicting directly over encrypted data, thanks to
+so-called Fully Homomorphic Encryption. A simple `Concrete ML`_ example would be as follows:
```

Comment:
No need for "so-called".

Probably just say "A simple example would be as follows".